Octokit.Tests.Integration/GithubClientExtensions.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Octokit.Tests.Integration
+{
+    public static class GithubClientExtensions
+    {
+        public static DisposableRepository2 CreateTestRepository(this IGitHubClient client, NewRepository newRepository)
+        {
+            return ((TestRepositoriesClient)client.Repository).CreateTestRepository(newRepository);
+        }
+    }
+
+    public class DisposableRepository2 : Repository, IDisposable
```

Comment:
I agree, it should be DisposableRepository. I had earlier created another class by the name DisposableRepository hence this class got its name as DisposableRepository2.